SPECS/libserf/libserf.spec

```diff
@@ -37,6 +37,21 @@ scons PREFIX=%{_prefix}
 scons PREFIX=%{buildroot}%{_prefix} install
 
 %check
+# The source tarball contains an expired cert, leading to 14 test failures. Skip these tests.
```

Comment:
Is it be possible to replace the trusted CA and the expired cert with a one issued by the new CA to enable the tests?

Reply:
I don't think we can do this without new certs provided from upstream. The most recent serf release was in 2016 and the certs expired in 2017.